docs/reference/hardware/powerovercoax.md

```diff
@@ -0,0 +1,70 @@
+
+![PoC-System-Diagram](../../images/PoC-System-Diagram.png)
+
+The inductive filters and power supply capacitance store energy during normal operation. The coax cable will charge up to the DC voltage of the power source such as +12 V when using A PXIe-148x's deserializer channel internal source and DC current will flow through the inductive filters. 
+
+The GMSL and FPD-Link standards do not support hot-plug or hot-disconnect capability. Connecting or disconnecting the coax cable while the power source is energized or even when there is still energy stored in the filters can lead to hardware damage.
+
+To prevent damage, turn off the power source and wait until the channel has discharged before disconnecting the coax cable. When using a PXIe-148x deserializer channel, turn the power source off by setting the FlexRIO Channel property `SI PoC Source` to `None`.
+
+![SI-PoC-Source-Property](../../images/SI-PoC-Source-Property.png)
+
+After turning off the power source, the electronics in the PXIe-148x and camera will dissipate the stored energy discharging the coax cable to a safe-to-disconnect state of 0 V. The amount of time it takes to reach the safe-to-disconnect state depends on the amount of stored energy and will vary between PXIe-148x and camera models. The FlexRIO LabVIEW Host API now has a function called **Wait for PoC safe to Disconnect** that helps determine when it is safe to disconnect.
+
+## Wait for Safe to Disconnect
+The **Wait for PoC safe to Disconnect** function waits and blocks until specified channels with power-over-coax are safe to disconnect to prevent accidental hardware damage. 
+
+### FlexRIO LabVIEW Host API
+The **Wait for PoC Safe to Disconnect** LabVIEW VI is located in the NI FlexRIO API Automotive Vision palette.
+
+![FlexRIO API Automotive Vision Palette](../../images/FlexRIO-API-Automotive-Vision-Palette.png)
+
+![Wait for PoC Safe to Disconnect VI](../../images/Wait-for-PoC-Safe-to-Disconnect.png)
+
+The key input parameters for the **Wait for PoC Safe to Disconnect** function are described below. See the LabVIEW context help for complete function documentation.
+
+The **Channels** input parameter accepts a string input type or an array of strings input type. When using the string input type, channels are specified as a single channel such as "SI0", a range such as "SI0-SI3" or a list such as "SI0,SI3". When using the array of strings input type, each element of the array is a single channel such as "SI0", "SO2", etc.
+
+The **Timeout (ms)** input parameter specifies a time limit in milliseconds to wait until safe to disconnect. A value of -1 indicates to wait indefinitely.
+
+> Warning: Waiting indefinitely causes the LabVIEW VI to hang if the discharge threshold for safe to disconnect is not met.
+
+> See the [Power Over Coax](../../reference/hardware/powerovercoax.md#setting-the-timeout) reference document for help in determining timeout values.
```

Comment:
This was copied from another file, but now it is referencing a section within the same file so the link should be updated.

```suggestion
> See [Setting The Timeout](#setting-the-timeout) for help in determining timeout values.
```